### DISCUSSION TITLE
Does this work when Obsidian is closed / device is offline?

### DISCUSSION THREAD

**Discussion by jamesvclements** [coddingtonbear/obsidian-local-rest-api#30] [Q&A] [answered]
Super intrigued by this plug-in. Does this only work when Obsidian is actively running on your device and your device is awake?
If so, is there any way to deploy it so that it runs online? Thinking this through I'm not sure how it would have access to your local vault if your vault isn't stored in the cloud somewhere, unless you can give permissions to a cloud function to access your device.

**Comment by coddingtonbear** [comment, marked as answer]
It doesn't, no -- this runs within Obsidian itself and provides an accessible API only while Obsidian is running.
